Assets/Engine/Inventory/UI/ContainerUi.cs

```diff
@@ -55,6 +56,25 @@ private void UpdateContainer(Container container)
             Vector3[] v = new Vector3[4];
             rect.GetLocalCorners(v); 
             containerName.transform.localPosition = v[1] + new Vector3(0.03f * width, -0.02f * height, 0);
+
+            if (attachedContainer.GetComponent<Item>() != null)
+            {
```

Comment:
Avoid calling `GetComponent` multiple times (save in variable first, then check against null).